[feature/new-navigation] New navigation

felix-schwarz opened the pull request:
## Description
Contains various changes and improvements to the view navigation and hierarchy of version 12:
- [x] sidebar
- [x] remove legacy table view code and use new UI for OC10, too
- [x] support for status 425 items
- [x] show folder statistics and drive quota (if one is set) at the bottom
- [x] show correct UI for ocis-based servers on first login for branded clients  
- [x] add missing German localizations
- [x] basic sharing fixes
- [x] new location picker, import sheet and share extension
- [x] show new [AppStore UI](https://developer.apple.com/documentation/storekit/appstore/3803198-showmanagesubscriptions) for managing AppStore subscriptions
- [x] browser-like navigation
- [x] tappable breadcrumb view

## Merges
This PR was started on a merge from #1142 and #1151 to leverage work from previous PRs directly.

In addition, the following PRs and changes have been merged into this branch:
- #1158 
- #1140 
- #1136 
- #1179
- merge `11.11.0` and `fastlane/enterprise-adhoc` changes from `master`

## Related Issues
#1161

## Screenshots (if appropriate):
Item that's processed on the server shown disabled | File unreadable in File Provider while processing on server
-|-
![Simulator Screen Shot - iPhone 14 - 2022-10-06 at 23 15 44](https://user-images.githubusercontent.com/639669/194424629-8acaef78-fcf6-41bd-8a6c-4f8ca8d4a6c4.png) |  ![Simulator Screen Shot - iPhone 14 - 2022-10-07 at 09 22 21](https://user-images.githubusercontent.com/639669/194496056-4e59a8d5-efa6-4369-9a4a-1f6324a9e0aa.png)

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] Added an issue with details about all relevant changes in the [**iOS documentation repository**](https://github.com/owncloud/docs-client-ios-app/issues).
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Added changelog files for the fixed issues in folder changelog/unreleased

## QA

Some sections with individual test plans for sub-features into this PR. Reports to be done separately (one report-one post), and grouped in this section for a better management.

Reports labeled with a severity approach: low, medium, high, critical

### Searching

Test plan: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/1142_search.md

- [ ] (20) [LOW] Soft keyboard over the results https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380055033
- [X] (21) [MED]  Hidden files in different scopes https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380071257
- [X] (22) [MED] Parent foler listed in `Tree` scope https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380079653 [FIXED]
- [X] (23) [LOW] Blank view if saved search returns no results https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380153233 [FIXED]
- [X] (24) [MED] Results displayed in wrong space https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380160779 [FIXED]
- [x] (25) [MED] Same search returns different results with `owner:` label https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380169520
- [X] (27) Concept of saved search and search template https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380186378 [FIXED]

### Sharing shortcuts in sidebar 

Test Plan: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/Sharing-oCIS.md

- [x] (1) (LOW) Accept/Decline buttons in iPhone / portrait. Thumbnail lost.  https://github.com/owncloud/ios-app/pull/1162#issuecomment-1375604403 [FIXED]
- [x] (2) [MED] Accepting share to blank view https://github.com/owncloud/ios-app/pull/1162#issuecomment-1375633114  [FIXED]
- [X] (3) [MED] Unsharing item from different client/device, leads to item in two statuses https://github.com/owncloud/ios-app/pull/1162#issuecomment-1376888919  [FIXED]
- [X] (4) Empty sections discordance in `Shared with me` https://github.com/owncloud/ios-app/pull/1162#issuecomment-1376946212 [FIXED]
- [X] (5) [HIGH] Resharing does not work https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377113675   [FIXED]
- [X] (6) [HIGH] Unshare does not work from "Shared by me" https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377168173 [FIXED]
- [X] (7) [HIGH] Not posible to share with user removing permissions https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377185130  [FIXED]
- [X] (8) [MED] Swipe action missing in `Shared by me` and `Shared by Link` https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377198049 [FIXED]
- [x] (9) [MED] `Copy` button missed in `Shared by Link` https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377234171 [FIXED]
- [X] (10) Link subtitle in `Shared by Link` section https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377284663 [WONT FIX]
- [X] (11) [HIGH] Remove link does not work from "Shared by link" https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377298241 [FIXED]
- [x] (36) [LOW] Saved searched gone after adding a search template https://github.com/owncloud/ios-app/pull/1162#issuecomment-1439651518 [FIXED]
- [X] (43) [HIGH] space_ref not sent in payload https://github.com/owncloud/ios-app/pull/1162#issuecomment-1488348322 [FIXED]
- [X] (46) [LOW] Copy button overridden by link https://github.com/owncloud/ios-app/pull/1162#issuecomment-1506459978 [FIXED]
- [x] (48) [MED] Arrows to open items missing in oC10 accounts https://github.com/owncloud/ios-app/pull/1162#issuecomment-1514475538 [FIXED]

### Quick access in sidebar

Test Plan: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/QuickAccess.md

- [X] (12) [CRIT] Crash when opening quick access item with iPhone https://github.com/owncloud/ios-app/pull/1162#issuecomment-1378397162 [FIXED]
- [x] (13) [LOW] Empty views in quick access https://github.com/owncloud/ios-app/pull/1162#issuecomment-1378490851 [FIXED]
- [x] (14) [HIGH] Av. offline shortcuts with not very good UX https://github.com/owncloud/ios-app/pull/1162#issuecomment-1378511094 [DONE]

### Spaces

Test Plan: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/1101_spaces.md

- [X] (15) [CRIT] Crash in Copy operation to a space https://github.com/owncloud/ios-app/pull/1162#issuecomment-1379970046 [FIXED]
- [x] (16) [CRIT] Crash when cancelling folder picker in copy operation https://github.com/owncloud/ios-app/pull/1162#issuecomment-1379972613 [FIXED]
- [X] (17) [LOW] Cancel button misplaced in Copy folder picker https://github.com/owncloud/ios-app/pull/1162#issuecomment-1379976962  [FIXED]
- [x] (18) [MED] Spaces list not adapted to iPhone screen width https://github.com/owncloud/ios-app/pull/1162#issuecomment-1379984735 [FIXED]
- [X] (19) [LOW] Lack of spaces empty view https://github.com/owncloud/ios-app/pull/1162#issuecomment-1379994327 [FIXED]
- [x] (28) [HIGH] `Open in a new window` not working https://github.com/owncloud/ios-app/pull/1162#issuecomment-1384983911 [FIXED]
- [X] (29) [MED] `Shares` option in folder picker https://github.com/owncloud/ios-app/pull/1162#issuecomment-1385071674 [RESOLVED]
- [X] (33) [MED] Options not available for `Viewer` level of permission https://github.com/owncloud/ios-app/pull/1162#issuecomment-1410298361 [FIXED]
- [x] (37) [MED] `Space` instead the space name on the top bar https://github.com/owncloud/ios-app/pull/1162#issuecomment-1439656519 [FIXED]
- [x] (44) [MED] Options's menu misplaced in spaces' list https://github.com/owncloud/ios-app/pull/1162#issuecomment-1497265379 [FIXED]
- [x] (45) [MED] List of spaces after closing account https://github.com/owncloud/ios-app/pull/1162#issuecomment-1504764246 [FIXED]
 
### IAP features
Scanner: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/Scanner.md
Markup: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/Markup.md
Shortcuts: https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%2012.0/Shortcuts.md

- [x] (32) [CRIT] Shortcuts feature broken https://github.com/owncloud/ios-app/pull/1162#issuecomment-1403340083 [DONE - personal space]
- [X] (42) [CRIT] Markup feature crashes in iPad https://github.com/owncloud/ios-app/pull/1162#issuecomment-1479063871
 [FIXED]
### Other

- [x] (26) [HIGH] Open in a new window missing https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380177357 [FIXED]
- [X] (30) [MED] Clickable area of 3-dot-button https://github.com/owncloud/ios-app/pull/1162#issuecomment-1385646876 [FIXED] 
- [X] (31) [HIGH] Not posible to share with oC from 3rd party apps https://github.com/owncloud/ios-app/pull/1162#issuecomment-1401558951 [FIXED] 
- [X] (34) [MED] Copy not available from preview https://github.com/owncloud/ios-app/pull/1162#issuecomment-1430964361 [FIXED]
- [X] (35) [HIGH] Move operation does not allow to select target folder in oCIS https://github.com/owncloud/ios-app/pull/1162#issuecomment-1438544879 [FiXED]
- [X] (39) [HIGH] Open in web does not open because bad language code https://github.com/owncloud/ios-app/pull/1162#issuecomment-1470096537 [FIXED]
- [X] (40) [MED] Operations' menu displayed in drag&drop https://github.com/owncloud/ios-app/pull/1162#issuecomment-1471483520 [NOT A BUG]
- [x] (41) [LOW] Some labels in german https://github.com/owncloud/ios-app/pull/1162#issuecomment-1477704246 [FIXED]
- [ ] (47) [LOW] Small accessibility issues https://github.com/owncloud/ios-app/pull/1162#issuecomment-1508151923

### iPad

- [ ] (38) [HIGH] Reopening shows empty list of accounts https://github.com/owncloud/ios-app/pull/1162#issuecomment-1469900811

**Missing Features:**

- Introduction view
- Themes: Dark Black, Dark Blue, Dark Web, Classic
- Favorites (server dependence)
- Private link opening
- Pull to refresh in file lists [DONE]
- Rearrange accounts list

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/owncloud/ios-app?pullRequest=1162) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/owncloud/ios-app?pullRequest=1162) before we can accept your contribution.<br/>**1** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: hosy<br/>:x: felix-schwarz<br/>:x: Matthias Hühne<br/><hr/>**Matthias Hühne** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/owncloud/ios-app?pullRequest=1162) it.</sub>

jesmrec commented:
### (1)

Upload a file from a different client, it's correctly shown as `Processing on server`. But, by clicking on it:

<img width="336" alt="Screenshot 2022-10-10 at 10 00 08" src="https://user-images.githubusercontent.com/14894746/194821564-db2ab063-d0e7-4e73-85c0-7fb6adbd16d8.png">

expected?

iPhoneXR, iOS16
`productversion  2.0.0-beta.8+6d32bafcf`

felix-schwarz commented:
@jesmrec Fixed (1) in [b6b9b91](https://github.com/owncloud/ios-app/pull/1162/commits/b6b9b9186ec97efc4f6d954135392866a4ac7b8d) with error message `{{itemName}} is currently processed on the server and can't be downloaded until it finishes processing.`.

![Simulator Screen Shot - iPhone 14 - 2022-10-10 at 12 05 31](https://user-images.githubusercontent.com/639669/194842450-f6a15ae7-7589-4373-9491-72c2d83c8be2.png)

jesmrec commented:
(1) fixed, elegant solution.

jesmrec commented:
Let's QA this

jesmrec commented:
### (1) (improvement)

**Feature**: Sharing
**Severity**: Low 

Steps to reproduce:

1. Share some items (including a picture) with another user in oCIS environment. (Personal space)
2. Recipient opens `Share with me` section in iPhone (portrait orientation)

Current: `Accept` and `Decline` options takes half-width of the screen, file name or file extension is cut off:

![Screenshot 2023-01-09 at 13 56 07](https://user-images.githubusercontent.com/14894746/211314180-ec72aa35-be68-43c1-803b-d5d73eacbdb9.png)

Suggestion: in iPhone, portrait orientation, showing only circle icons and avoid words.

iPhoneXR v16.0.2
`c45ad917e`

jesmrec commented:
### (2) (maybe etag-server related)

Feature: Sharing
Severity: Medium

Steps to reproduce:

1. User1 shares item with User2 (personal space)
2. User2 accepts share in `Share with me` section

Current: `Share with me section` becomes empty and blank. No matter how long you wait:

https://user-images.githubusercontent.com/14894746/211319154-ea80b14c-0f53-4a31-9d12-11bd7922418f.MP4

By browsing back and forth, everything correct.

Expected: Accepted item moved to `Accepted`. This action should be triggered by the clicking action. Users' view should never be blank

iPhoneXR v16.0.2
`c45ad917e`

jesmrec commented:
### (3)

Feature: Sharing
Severity: Medium

Steps to reproduce:

1.  User1 shares item with User2
2. User2 accepts the item
3. in a different client, User2 unshares the item by swiping left and clicking in `Unshare`

Current: item is both `Accepted` and `Declined`:

<img width="333" alt="Screenshot 2023-01-10 at 09 18 25" src="https://user-images.githubusercontent.com/14894746/211497694-7152043d-a9ce-46bd-ab8e-e2f16896ced8.png">

by browsing back and forth, it refreshes to the correct status.

Expected: one item in only one status

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (4) (question)

I notice that the three sections in `Share with me` are sometimes listed and sometimes not listed with and without items inside. Any reason? does it depends on the API retrieved sections?

|||
|:-:|:-:|
|<img width="329" alt="Screenshot 2023-01-10 at 09 47 23" src="https://user-images.githubusercontent.com/14894746/211503918-f12902ae-c614-4c51-9612-c00512aea4ec.png">|<img width="332" alt="Screenshot 2023-01-10 at 09 55 10" src="https://user-images.githubusercontent.com/14894746/211505753-9225a095-fc9e-45fa-bc0c-f22f5b52f253.png">|
|<img width="330" alt="Screenshot 2023-01-10 at 09 49 02" src="https://user-images.githubusercontent.com/14894746/211504372-eb7cf075-2e50-415b-a971-67ade5666ffe.png">|<img width="329" alt="Screenshot 2023-01-10 at 09 51 24" src="https://user-images.githubusercontent.com/14894746/211504850-9ebd1d03-a9c5-4910-882b-1f22520383a3.png">|

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (5)

Feature: **Sharing**
Severity: **High**

Steps to reproduce:

1. User1 shares item with User2 
2. User2 accepts the share
3. In `Shared with me` section, User2 tries to share the item with User3 (User2 is granted to do that)

Current: error `Wrong path, file/folder doesn't exist`
Expected: no error. Operation allowed. Web client completes action with no error. Comparing request sent by iOS and web client, there are differents in the request payload

iOS:

```
path: /land6.png
shareType:   0
permissions: 17
shareWith:   user3
```

web:
"shareType": "0",
"shareWith": "user3",
"path": "/land6.jpg",
"space_ref": "c3e9283b-642f-407b-94da-dfdfd057dc85$f4cc5b14-7bf2-45c1-b470-e3d94969083e!947b6b85-3f75-4761-b665-801792cfbbf8",
"permissions": "17",
"role": "viewer"
}

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (6) (maybe server involved)

**Feature**: Sharing
**Severity**: High

Steps to reproduce:

1. User1 shares item with User2
2. User1 opens `Shared by me`
3. User1 swipes item to delete share , clicking the red zone

Current: error `grpc update received share request failed`
Expected: item unshared. 

`DELETE` request returns `500` (server involved) with the following payload:

```
<?xml version="1.0" encoding="UTF-8"?>
<ocs>
  <meta>
  <status>error</status>
  <statuscode>996</statuscode>
  <message>grpc update received share request failed</message>
  </meta>
</ocs>
```
Web client sends nothing relevant in request payload (undefined: undefined) and action is correctly done there. iOS sends the share_id

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (7)

**Feature**: Sharing
**Severity**: High

Steps to reproduce:

1. Open Personal space
2. Share file with another user
3. Remove all permissions: `Can Share` and `Can Edit and Change` disabled
4. `Save`

Current: error `Cannot set the requested share permissions`
Expected: file shared

Request is sent from iOS app with `permissions: 4` . It should be sent with `permissions: 1`, right? 

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (9)

**Feature**: Sharing
**Severity**: Medium

in iPad (iPadOS15), the `Copy` button is missed in `Shared by Link` section. A `Copy` text in white font is placed instead:

![Screenshot 2023-01-10 at 14 01 12](https://user-images.githubusercontent.com/14894746/211559889-c479548d-7280-472b-97aa-90cc8f78bb0c.png)

iPadAir v15
`c45ad917e`

jesmrec commented:
### (10) (question)

In the `Shared by Link` section, the link is displayed just below the filename together with the file name (check screenshot in report 9)

IMO, that subtitle shows no relevant info, since the file name is the cell main info, and the link itself lacks of meaning.

Other possibilities:

- Show nothing as subtitle
- Show size and date, as the file list
- Show if link is protected (lock icon), and expiration date 

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (11)

**Feature**: Sharing
**Severity**: High

Steps to reproduce:

1. Share an item by link
2. In `Shared by Link` section, swipe left or long press on the item to unshare

Current: error `Insufficient parameters`
Expected: link removed

no `DELETE` request is sent

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (12)

Feature: Quick Access
Severity: Critical

Using iPhone

1. Open Quick Access section
2. Open any subsection with items
3. Click on the right arrow that points to the file

Current: app crashes
Expected: no crash

NOTE: only reproducible with iPhone. iPad does not.

iPhoneXR v16.0.2
`c45ad917e`

jesmrec commented:
### (13)

**Feature**: Quick Access
**Severity**: Low

Empty views in quick access with no items

| App Store | Current |
|:-:|:-:|
|![Screenshot 2023-01-11 at 10 49 56](https://user-images.githubusercontent.com/14894746/211773786-03f27e3b-4f87-4826-9d88-0bb804426e99.png)|<img width="335" alt="Screenshot 2023-01-11 at 10 44 17" src="https://user-images.githubusercontent.com/14894746/211772870-055fb905-847c-4052-82e5-922b9da7eb50.png">|

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (14)

**Feature**: Quick Access
**Severity**: High

This is how the av. offline shortcut section looks like: 

![Screenshot 2023-01-11 at 11 00 46](https://user-images.githubusercontent.com/14894746/211776909-77b9fe8a-fdae-4f24-9cdd-53eb6d8d8858.png)

There are two lists. What's the upper list used for? i realised that an av. offline folder appears only in the upper list and its content in the list below. But an av. file appears in both. That's really confusing.

The `11.11.1` versions lists only those items that were set as av. offline by the user, in case of folders just the folder and not the content (implicitly av. offline). I'd go for that approach, that matches with the content of the upper list in the current. What do you think?

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (15)

**Feature**: Spaces 
**Severity**: Critical

Steps to reproduce:

1. Enter in personal space
2. Select `Copy` and `Choose destination directory`
3. Select as destination an existing space (not personal)

Current: crash
Expected: file copied 

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
###  (16)

Feature: Spaces
Severity: Critical

Steps to reproduce:

1. Enter in personal space
2. Select Copy and Choose destination directory
3. Cancel in folder picker

Current: crash
Expected: view dismissed

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
###  (17)

**Feature**: Spaces
**Severity**: Low

Folder picker in `Copy` operation shows `Cancel` button misplaced in iPhone:

![Screenshot 2023-01-12 at 09 31 36](https://user-images.githubusercontent.com/14894746/212017256-278fc1cf-23d7-410c-96df-0e84d56183de.png)

iPhoneXR v16.0.2
`c45ad917e`

jesmrec commented:
### (18)

**Feature**: Spaces
**Severity**: Medium

Spaces' view shows 3 items per row. This works very fine in iPad, but the reduced width in iPhone is not so good:

<img width="304" alt="Screenshot 2023-01-12 at 09 38 43" src="https://user-images.githubusercontent.com/14894746/212018729-376c02e6-1163-4566-b4ee-a93623c3743f.png">

I'd set 2 per row in that case.

iPhoneXR v16.0.2
`c45ad917e`

jesmrec commented:
### (19)

**Feature**: Spaces
**Severity**: Low

Steps to reproduce:

1. Add an oCIS account with no spaces (or remove all spaces of an account)
2. Open `Spaces` from side bar

Current: blank view
Expected: empty view indicator, like in web:

<img width="467" alt="Screenshot 2023-01-12 at 09 49 55" src="https://user-images.githubusercontent.com/14894746/212020805-d76c7a99-567f-45a2-87c1-d8be16847638.png">

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (20)

**Feature**: Searching
**Severity**: Low

In a folder with some files (more than 10 for iPhone, more than 20 for iPad). 

1. Click on lens icon
2. Select `Folder` as scope
3. Select `Type = file` 

Current: all files in the folder are correctly listed. But, the files on the screen bottom are hidden behind the soft keyboard
Expected: any way to vanish the soft keyboard?

https://user-images.githubusercontent.com/14894746/197511405-cf9646e8-020b-4426-b0f0-13a3d12e8ea2.MP4

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (21)

In an empty space (no matter if personal or custom space):

1. Click on lens icon
2. Set `Folder` scope
3. Set `Type = Folder` -> empty result
4. Set `Tree` scope
5. Set `Type = Folder` 

Current: hidden folders like `.space` and `/` (root) appears as result in tree but not in folder. 
Expected: same results for the current folder in both scopes

https://user-images.githubusercontent.com/14894746/197519923-e875faaf-80a1-477d-ac2b-4a38b85f50ed.MP4

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (22)

**Feature**: Searching
**Severity**: Medium

(probably related with (21)

1. Create a folder structure like `A/B/C`
2. Browse to `B`   (`C` is listed)
3. Click on lens
4. Set `Tree` as scope
5. Set `Type = folder` 

Current: Results are `B`(current folder) and `C`
Expected: Result is `C`,  because `B` is out of the tree (parent folder). If expectation is different just let me know which one is expected.

https://user-images.githubusercontent.com/14894746/197522342-178af8a0-a3db-4526-af3d-a9e4cc9ed10e.MP4

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (23)

**Feature**: Searching
**Severity**: Low

1. Create a `Saved Search` that returns empty result list
2. Save it, and go back to the list of files
3. Click on lens icon and apply that `Saved Search`

Current: empty screen
Expected: `No matches` view displayed

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (24)

**Feature**: Searching
**Severity**: Medium

1. Create an empty space, called `Space1`
2. Create another empty spaced called `Space2`
3. Upload two videos to `Space2`
4. Open `Space1`
5. Click on lens icon and select `Account` scope
6. Select `Type = Video` -> Videos in `Space2` are shown as result :+1: 
7. Click on right arrow, at the right of the result's cell, to jump to the file location

Current: `Space1` shown with the videos. That's not correct because videos are stored in `Space2`
Expected: Videos stored in `Space2`, so `Space2` should be displayed instead

(name of spaces and videos as type are only examples, also reproducible with other options)

https://user-images.githubusercontent.com/14894746/198285891-bbddb485-a7a7-4ed7-9415-184f9844a4f8.MP4

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
(25)

Feature: Searching
Severity: Medium

This one is about the `owner:` label . That seems to work only in personal space.

Steps:

1. User1 uploads a file to personal space 
2. User1 uploads same file to another space 
3.  User1 searchs `owner:User1` in personal space with folder scope -> it returns the uploaded file (and others, if exist un the current folder)
4. User1 searchs `owner:User1` in the other space with folder scope -> it returns nothing

Expected: same result in both 3. and 4. , because owner is the same.

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (26)

**Feature**: Global
**Severity**: High

in iPad, option `Open in a new window` does nothing (pending to re-develop?)

iPadAir v15
`c45ad917e`

jesmrec commented:
### (27) (question)

**Feature**: Searching

Mentioned in https://github.com/owncloud/ios-app/pull/1142#issue-1351408607:

> saved searches save scope (folder / account / space), filters (type, date, size, …), search terms and the searched location (think "Smart Folders" like on macOS)
search views save scope (folder / account / space), filters (type, date, size, …), search terms, but not the searched location (think "search settings quickly re-applicable to any location")

is that correct?

Checking the implementation, i noticed the opposite behaviour

- `Saved search template` applies the filter to the current location. For ex, `Type = Tree "GIF"` returns a folder called `GIF` in the current folder and its subfolders. Same `Saved search template` applied to other folder with no matches, returns empty. That means, searched location not saved.

https://user-images.githubusercontent.com/14894746/198271484-265b17f0-dad5-4600-b710-062521805b4f.MP4

- `Saved search view`, returns always the same result, no matter where the filter is applied:

https://user-images.githubusercontent.com/14894746/198273616-fe6b3e71-c048-4f05-a538-ce0f79a60015.MP4

Which is correct, the implementation or the description? 

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (28)

**Feature**: Spaces
**Severity**: High

`Open in a new window` is not working:

https://user-images.githubusercontent.com/14894746/212841261-7e65e4f8-53b0-4bfe-9dff-bf1ef802baec.MP4

iPadAir v15
`c45ad917e`

jesmrec commented:
### (29)

**Feature**: Spaces
**Severity**: Medium

1. Inside any space, select an item to `Copy`

this is the look of the new folder picker:

![Screenshot 2023-01-17 at 10 13 48](https://user-images.githubusercontent.com/14894746/212857603-b0737c32-7051-4ec0-89fa-095dd763fac2.png)

`Shares` project should not be an option to copy/move in. Just the spaces containing files

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (30)

**Feature**: Other
**Severity**: Medium-High

Clickable area of three-dot-button is now pretty reduced in comparison with previous version. It leads users to mistakes, downloading files that were not intended to by clicking on the cell:

|11.11.1 | Current |
| :-: | :-: |
|![Screenshot 2023-01-17 at 16 54 55](https://user-images.githubusercontent.com/14894746/212947312-b7ff84d1-c22c-4427-8d75-b51512343373.png)|![Screenshot 2023-01-17 at 15 13 27](https://user-images.githubusercontent.com/14894746/212947276-fba133f2-21d7-4b6f-bc61-a4fe8e713778.png)

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (31)

**Feature**: Other
**Severity**: High

1. Open `Photos` app
2. Select some pictures
3. Share them with oC

Current: Pictures not uploaded. No matter if the account is oC10, oCIS or the kind of space.
Expected: Pictures uploaded.

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (32)

**Feature**: Other
**Severity**: High

Shortcuts IAP feature is broken. 

- Actions require `Path` parameter, and needs to adapt to the new structure with spaces. Selecting space in another parameter could be a solution but only if account is oCIS. This is affecting almost all actions

- `Get Accounts` don't retrieve the list of active accounts

iPhoneXR v16.0.2, iPadAir v15
`c45ad917e`

jesmrec commented:
### (33) 

**Feature**: Spaces
**Severity**: Medium

1. Create a new space in web
2. Add a new member with `Viewer` permission
3. In app, viewer member open the space he/she was added to

Current:

- `+` option in top bar is available, even when no options inside are available (viewer not granted to add new content)
- `Cut` option available in actions card. `Cut` option is destructive (removes source file after pasting) and should not be available.

Expected:

Neither `+` nor  `Cut` should be available for users with `Viewer` permission

iPhoneXR v16.0.2, iPadAir v15
`bd70cc29`

felix-schwarz commented:
@jesmrec Findings 12, 26 and 28 should be fixed as of https://github.com/owncloud/ios-app/pull/1162/commits/fa025d0a879730edd0461a7e05ca2140921eb0a7 (build 248)

jesmrec commented:
12 -> fixed
26, 28 -> open in new window. It works fine, but sometimes a little lag happens before showing the content (i could attach a video), or the list of accounts is displayed before. As enhancement, setting a waiter or spinner could help to unpatient users. But the initial problem is fixed ftm.

> Re (29): why should those items no be available there? I'd think that copying a file into a folder shared with or by me should be supported.

for the "shared with me", makes sense. If a folder is shared with writing permissions, it's the way to upload something there. If the folder is "shared by me" or "shared by link", it could be selected in the main location (personal or space). But ok, it does not hurt.

In any case, if copying inside the "Shares" space is allowed for "shared with me" folders, it should be only for the "Accepted" section. "Pending" or "Declined" shares are listed in the Copy picker, and copy inside a folder in such status makes no sense. What do you think @felix-schwarz?

jesmrec commented:
### (34)

**Feature**: Other
**Severity**: Medium

1. Preview any file (image, PDF...) in any space, does not matter
2. Click on three-dot-button and click on `Copy`
3. `Choose destination directory...`

Current: nothing happens
Expected: folder picker with the list of accounts/spaces displayed

iPhoneXR v16.3
`fa025d0a`

jesmrec commented:
### (35)

**Feature**: Other
**Severity**: High

1. in a oCIS account, select file and operation `Move` -> Folder picker displayed with content of the current space
2. Select any folder as target for the `Move`

Current: not able to browse
Expected: able to browse

iPhoneXR v16.3
`fa025d0a`

felix-schwarz commented:
@jesmrec wrote:

> 26, 28 -> open in new window. It works fine, but sometimes a little lag happens before showing the content (i could attach a video), or the list of accounts is displayed before. As enhancement, setting a waiter or spinner could help to unpatient users. But the initial problem is fixed ftm.

The most obvious way to address that would be to cover the actual view with a progress indicator until the new scene has been fully built. The challenge then, however, is how to capture and present errors - direct or indirect - that spring from building the requested scene in a new window. Like authentication issues, etc. There's a risk they, too, would be covered.

Still, I'll add this to the `Evolution` part of `KNOWN_ISSUES.md` for when there is more time available to take a closer look at implementing this.

> > Re (29): why should those items no be available there? I'd think that copying a file into a folder shared with or by me should be supported.
> 
> for the "shared with me", makes sense. If a folder is shared with writing permissions, it's the way to upload something there. If the folder is "shared by me" or "shared by link", it could be selected in the main location (personal or space). But ok, it does not hurt.
> 
> In any case, if copying inside the "Shares" space is allowed for "shared with me" folders, it should be only for the "Accepted" section. "Pending" or "Declined" shares are listed in the Copy picker, and copy inside a folder in such status makes no sense. What do you think @felix-schwarz?

I think having the full sharing section here can still be useful.

Imagine a user receiving a call from a coworker or customer, whereby the user is asked to copy a file to a shared folder.

Even the ability to accept a declined or pending share can be useful in that situation, if f.ex. the caller is sharing the target folder while the user is already in the copy UI.

felix-schwarz commented:
@jesmrec (13), (14) and (23) should be fixed as of https://github.com/owncloud/ios-app/pull/1162/commits/13a0d385231dbae57b00e83865f7914db6786cf8

jesmrec commented:
> The most obvious way to address that would be to cover the actual view with a progress indicator until the new scene has been fully built. The challenge then, however, is how to capture and present errors - direct or indirect - that spring from building the requested scene in a new window. Like authentication issues, etc. There's a risk they, too, would be covered.

Still, I'll add this to the Evolution part of KNOWN_ISSUES.md for when there is more time available to take a closer look at implementing this.

Thanks, i will set 26 and 28 as fixed.

> I think having the full sharing section here can still be useful.

Imagine a user receiving a call from a coworker or customer, whereby the user is asked to copy a file to a shared folder.

Even the ability to accept a declined or pending share can be useful in that situation, if f.ex. the caller is sharing the target folder while the user is already in the copy UI.

Let's give a try. We are always on time to improve and iterate existing behaviours. Will set (29) as resolved

jesmrec commented:
> @jesmrec (13), (14) and (23) should be fixed as of https://github.com/owncloud/ios-app/commit/13a0d385231dbae57b00e83865f7914db6786cf8

all of them, fixed!

jesmrec commented:
### (36)

**Feature**: Search
**Severity**: Low

Small glitch

1. Add new account -> In sidebar, option `Saved searches` is visible and empty
2. Open any space, do any search
3. Save as `search template`
4. Open sidebar

Current: `Saved searches` is gone
Expected: `Saved searches` is visible and empty. Same as the step 1. after adding the account 

iPhoneXR v16.3
`fa025d0a`

jesmrec commented:
### (37)

**Feature**: Spaces
**Severity**: Medium

Open the root of any space, no matter if personal or any other one. On the top, `Space` instead of the space's name:

![Screenshot 2023-02-22 at 10 02 38](https://user-images.githubusercontent.com/14894746/220572372-9bff6c22-8775-493c-a2fa-b50700608d09.png)

IMO, there should be the space name.

iPhoneXR v16.3
`13a0d385`

felix-schwarz commented:
@jesmrec (4), (19), (21), (22), (24) should be fixed as of https://github.com/owncloud/ios-app/pull/1162/commits/62991a48b0630b36b081b5c3d02b312bd51f5a61

felix-schwarz commented:
@jesmrec (36) should be fixed as of https://github.com/owncloud/ios-app/pull/1162/commits/0a60d26a64a161aa1eb31b6bb617b95475fb061c

jesmrec commented:
(4) -> fixed. I noticed a small glitch that i will report in `Sharing shortcuts in sidebar`
(22) -> still reproducible (`0a60d26a`)
(19), (21), (24), (36) -> fixed

jesmrec commented:
### (38)

**Feature**: iPad
**Severity**: High

1. Install the app from scratch to iPad
2. Add some accounts
3. Close the app
4. After some minutes open the app

Current: Attached accounts are not there, empty view. By clicking on `+` and adding new account, all the accounts appear again

Expected: list of accounts visible after reopening

Video recording with the `Current` section (app reopened, before the accounts were attached). You can see in the video that in the opening there is no accounts (just the `+`) and after adding new account, the three accounts attached before appear again

https://user-images.githubusercontent.com/14894746/225292511-4790d558-e1fe-4f1e-a7ff-31518386f9ff.MP4

Logs caught: [ownCloud_15_Mar_2023_at_11_55_41.log.txt](https://github.com/owncloud/ios-app/files/10978954/ownCloud_15_Mar_2023_at_11_55_41.log.txt)

iPadPro, iPadOS16.3 
`de4381b5`

jesmrec commented:
### (39)

Feature: Other
Severity: Medium

1. Long press on any item of the file list

Current: Multiselection's operations menu in the bottom side is displayed

Expected: Multiselection's operations menu in the bottom side is not displayed, because operations are not selectable in this scenario.

https://user-images.githubusercontent.com/14894746/225552694-a34e4f40-1043-4cd9-8e4b-3fdf278b37d3.MP4

iPhoneXR, 16.3
`60f756ee`

felix-schwarz commented:
@jesmrec I fixed (35).

felix-schwarz commented:
@jesmrec I fixed (8) (11) (17) (22) (30) (34) (35) (39).

I hope to have fixed (38) (couldn't reproduce, but had an idea of what could cause it - and fixed that).

---

I couldn't reproduce (9), but this purely looks like a theming issue. Does it still exist with the latest `feature/new-navigation`?

---

I also couldn't reproduce (37). Do you still see this issue with the latest `feature/new-navigation`?

---

In (40), what you see are not the multiselect actions, but the drop targets that appear when you start dragging. They are there to drop items onto.

---

Re (27): `search view` was renamed to `search template`, so now that would read:
> - `saved searches` save scope (folder / account / space), filters (type, date, size, …), search terms and the searched location (think "Smart Folders" like on macOS)
> - `search templates` save scope (folder / account / space), filters (type, date, size, …), search terms, but *not* the searched location (think "search settings quickly re-applicable to any location")

And that's also how it currently is implemented.

---

Re (10): To see the link could be useful if a user wants to delete a particular link. It is also shown in the web UI. ![Bildschirm­foto 2023-03-18 um 16 40 46](https://user-images.githubusercontent.com/639669/226115765-a47f253e-55e0-401c-9d2d-37075e890836.png)

jesmrec commented:
`app:1e87b9ed, sdk: 3010676e, build 252`

> @jesmrec I fixed (8) (11) (17) (22) (30) (34) (35) (39).

all of them fixed, cool!

> I hope to have fixed (38) (couldn't reproduce, but had an idea of what could cause it - and fixed that).

still reproducible. More logs caught with the latest build:

[ownCloud_21_Mar_2023_at_12_07_32.log.txt](https://github.com/owncloud/ios-app/files/11027669/ownCloud_21_Mar_2023_at_12_07_32.log.txt)

> I couldn't reproduce (9), but this purely looks like a theming issue. Does it still exist with the latest feature/new-navigation?

reproducible only in iPadOS15. `Copy` option only visible by long pressing on the cell. Let me know in more details are required

> I also couldn't reproduce (37). Do you still see this issue with the latest feature/new-navigation?

yes, it seems to happen in the first navigation through the spaces in a recent attached account. Check the following video, by adding a new account and opening the existing spaces, the top header is `Spaces`:

https://user-images.githubusercontent.com/14894746/226575976-5d1c5b7d-36c3-4d88-a9c0-a45c9aeeb535.MP4

> In (40), what you see are not the multiselect actions, but the drop targets that appear when you start dragging. They are there to drop items onto.

Thanks for clarification 👍 

> Re (27): search view was renamed to search template, so now that would read:

done and solved

> Re (10): To see the link could be useful if a user wants to delete a particular link. It is also shown in the web UI.

It's OK for me. This does break nothing and if it is useful in some cases, we can keep it

jesmrec commented:
### (41)

Feature: **Other**
Severity: **Low**

Click on `+` and `New Document` in oCIS account with app providers:

<img width="336" alt="Screenshot 2023-03-21 at 12 52 17" src="https://user-images.githubusercontent.com/14894746/226598197-89bb8c6f-abb7-494e-9dbf-bce15e0e3c5c.png">

text in German by default, with device in English

iPhoneXR iOS16
`1e87b9ed`

jesmrec commented:
### (42)

**Feature**: iPad, IAP
**Severity**: Critical

Reproducible with iPad (16)

1. Open the card menu of any image and select `Markup`

Current: file is downloaded and previewed. When markup mode is enabled, app crashes

Expected: no crash

https://user-images.githubusercontent.com/14894746/226836928-3d873f6a-baf5-417c-a3eb-5a5cdb5e7312.MP4

Some logs:  [ownCloud_22_Mar_2023_at_08_53_12.log.txt](https://github.com/owncloud/ios-app/files/11037658/ownCloud_22_Mar_2023_at_08_53_12.log.txt)

iPadPro iPadOS16
`1e87b9ed`

jesmrec commented:
Reviewed the list of pending reports in build 252, report (6) is also fixed with the latest changes

At this point: `app:1e87b9ed, sdk: 3010676e, build 252` , the most significative detected remaining problems are: 

- [(31)](https://github.com/owncloud/ios-app/pull/1162#issuecomment-1401558951): Not posible to share from 3rd party apps
- [(42)](https://github.com/owncloud/ios-app/pull/1162#issuecomment-1479063871): Markup crashes for images (iPad)
- [(32)](https://github.com/owncloud/ios-app/pull/1162#issuecomment-1403340083): Shortcuts feature broken
- [(38)](https://github.com/owncloud/ios-app/pull/1162#issuecomment-1469900811): Empty list of accounts (iPad)
- [(7)](https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377185130): Not posible to share without permissions

The other reports are there but their impact in the functionality is not too high. Mainly glitches and corner case that needs some attention and should be fixed with less priority than the listed ones above. 

New round will be done as soon as new features included.

jesmrec commented:
### (43)

**Feature**: Other
**Severity**: High

Create a new oCIS server adding the environment variable `PROXY_ENABLE_BASIC_AUTH: "true"`. This allows to use basic auth n oCIS, needed for QA purposes.

That server will have both OIDC and basic auth enabled

Then, install and open the app, and connect the recently created server. 

Current: Prompt to basic auth 
Expected: Prompt to OIDC (default)

iPhoneXR iOS16
`1b998941`

felix-schwarz commented:
@jesmrec (9), (18) and (37) are fixed in `feature/css`.

(41) might be fixed as well in `feature/css`, because language codes submitted to the server when querying for available document formats and apps are now formatted in the format expected by the server. If it still isn't, please capture a log and share it with me.

Where can I find info on `[HIGH] Basic auth prompted when OIDC is available`? It's listed as (43) in the top list without a working link, but in the comments (43) is a sharing issue.

jesmrec commented:
> Where can I find info on [HIGH] Basic auth prompted when OIDC is available? It's listed as (43) in the top list without a working link, but in the comments (43) is a sharing issue.

i removed this one because because it's not an app problem, i caused it in the test environment by changing the order of authentication methods with testing purposes. Sorry, i will remove from lists as well.

jesmrec commented:
> Where can I find info on [HIGH] Basic auth prompted when OIDC is available? It's listed as (43) in the top list without a working link, but in the comments (43) is a sharing issue.

i removed this one because because it's not an app problem, i caused it in the test environment by changing the order of authentication methods with testing purposes. Sorry, i will remove from lists as well. Number (43) corresponds to another different report

jesmrec commented:
(9), (18), (37) and (41) fixed

jesmrec commented:
### (44)

**Feature**: Spaces
**Severity**: Medium

If a space has a long name, the three-dot-button menu is misplaced in the list in both orientations:

<img width="336" alt="Screenshot 2023-04-05 at 12 26 12" src="https://user-images.githubusercontent.com/14894746/230054872-d3e8edd1-13a1-4311-a759-9df5dd2ba5c1.png">

<img width="640" alt="Screenshot 2023-04-05 at 12 28 12" src="https://user-images.githubusercontent.com/14894746/230054986-3017e38a-c28f-49f1-b6b5-e47ef856af0f.png">

iPhoneXR v16
`5a460dc2`

jesmrec commented:
### (45)

**Feature**: Spaces
**Severity**: Medium

Reproducible with iPad (landscape mode):

1. Add an account with spaces
2. In sidebar menu, click on `Spaces` -> list of spaces in screen
3. In sidebar menu, click on `Close account`

Current:

list of spaces in screen

Expected:

List of accounts in the screen

iPadAir v15
`5a460dc2`

felix-schwarz commented:
@jesmrec (45) is fixed by https://github.com/owncloud/ios-app/pull/1195/commits/d8ebfd07d5ef8ccb59607617481e9e53eff1c98c in https://github.com/owncloud/ios-app/pull/1195

jesmrec commented:
### (46)

**Feature**: Sharing
**Severity**: Low

Maybe related with the fix [(9)](https://github.com/owncloud/ios-app/pull/1162#issuecomment-1377234171)

Now, the `Shared by link` section looks like this:

<img width="337" alt="Screenshot 2023-04-13 at 09 05 52" src="https://user-images.githubusercontent.com/14894746/231680497-214b15b6-fcee-47d2-a163-656f6107fd20.png">

the link is overridden by the `Copy` button

iPhone XR v16
`da208f55`

jesmrec commented:
### (47)

**Feature**: Sharing
**Severity**: Low

After checking all new items in 12.0.0 in VoiceOver dictation, two things to comment:

- I was not able to perform a long press. Long press gesture is useful in two ways:
   - Drag & Drop in the list of files
   - Open the edit, manage menu of accounts in the sidebar
- Status view: `All done` and subtitle are not spelt

iPhoneXR iOS16
`862ea933`

jesmrec commented:
### (48)

**Feature**: Sharing
**Severity**: Medium

1. In oC10 account, open the "Shared with me" section (some items there in advance)

Current: Arrow to open items missing
Expected: Arrow to open items there

iPhoneXR iOS16
`955b38a6`

felix-schwarz commented:
@jesmrec Regarding the current state of `fix/v12-fixes`:

- (25): I noticed that `owner:` searches for user names starting with the provided user name, not the exact user name and fixed that. It now needs to match exactly. Other than that I noticed that `<oc:owner-id>` and `<oc:owner-display-name>` are not returned for `PROPFIND`s  on spaces, so that search can't match any owner inside spaces currently. As per the ocis team, this is the expected behaviour ("there is no owner in spaces").

- (31) (33) (46) are fixed

jesmrec commented:
> (7) (31) (33) (43) (46) are fixed

(7) (31) (33) (43) (46)

> (5) re-sharing currently only works if the shared file is inside a space the user also has access to. This is a consequence of a lack of/uniqueness of information (https://github.com/owncloud/ocis/issues/5355) and therefore can't be addressed at this time.

Let's wait for the backend fix. Then, resharing will be checked again

jesmrec commented:
> (25): I noticed that owner: searches for user names starting with the provided user name, not the exact user name and fixed that. It now needs to match exactly. Other than that I noticed that <oc:owner-id> and <oc:owner-display-name> are not returned for PROPFINDs on spaces, so that search can't match any owner inside spaces currently. As per the ocis team, this is the expected behaviour ("there is no owner in spaces").

about this one, providing exact username in both mentioned searchings in the report, just the `Personal` one returns the expected result. Check the following video: in both `Personal` and `space1` there is video file. Same search in both returns the file in `Personal` and nothing in `space 1`: 

https://user-images.githubusercontent.com/14894746/234811100-50d7cf8e-fad4-4dac-9084-348243d799b8.MP4

felix-schwarz commented:
@jesmrec Regarding (25):

> about this one, providing exact username in both mentioned searchings in the report, just the `Personal` one returns the expected result. Check the following video: in both `Personal` and `space1` there is video file. Same search in both returns the file in `Personal` and nothing in `space 1`:

This is not a bug. Searches for owners in spaces other than the Personal space will not return any results right now because spaces other than the Personal space do not provide owner information for their items.

felix-schwarz commented:
@jesmrec (5) is fixed in `fix/v12-fixes`.

jesmrec commented:
> @jesmrec (2) (3) (5) (42) (48) is fixed in fix/v12-fixes.

all fixed

felix-schwarz commented:
@jesmrec (1) is now also fixed in fix/v12-fixes

jesmrec commented:
(1) fixed

For (47):

In device Settings > Accessibility, enable VoiceOver

First scenario:

1. Open oC app, any account
2. In the list of files, try to drag any item by long pressing -> not posible (at least, i did not find a way. maybe not allowed with VoiceOver?)

Second scenario:

1. Open sidebar and select `Status` of any account which `Status` list is empty
2. Click on `All done` message -> no dictation. Other empty lists have dictation

jesmrec commented:
Pending stuff:

[HIGH]:

(38) Accounts list lost in iPad https://github.com/owncloud/ios-app/pull/1162#issuecomment-1469900811

Some logs attached here: https://github.com/owncloud/ios-app/pull/1162#issuecomment-1477688916

[LOW]:

(47) Accessibility issues https://github.com/owncloud/ios-app/pull/1162#issuecomment-1508151923
(20) Soft keyboard over the results https://github.com/owncloud/ios-app/pull/1162#issuecomment-1380055033

felix-schwarz commented:
Merged into https://github.com/owncloud/ios-app/pull/1095.
